scripts/run-indexer.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Same, need docs on how to run

Reply:
@tomerlichtash 
(also relevant to other script(s))
1. You don't need to run the scripts, they're not standalone. You only run `yarn build-index` (or `yarn dev-index`).
2. The scripts are documented and print a `usage` message when run with no params